### PR TITLE
Add ENV_CONTROLLER_API_URL for run-time controller configuration

### PR DESCRIPTION
**Pull request by apasyniuk**
Add new parameter for run-time configuration of controller agent API URL.

### CODE REVIEW

File: amazon/no-nap/entrypoint.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #
 # This script launches nginx and the NGINX Controller Agent.
 #
-# If several instances use the same imagename, the metrics will
+# If several instances use the same image name, the metrics will
```
Comment (brianehlert) [inline amazon/no-nap/entrypoint.sh:5]: image name is no longer a thing.  That was legacy from amplify

File: centos/no-nap/entrypoint.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #
 # This script launches nginx and the NGINX Controller Agent.
 #
-# If several instances use the same imagename, the metrics will
+# If several instances use the same image name, the metrics will
```
Comment (brianehlert) [inline centos/no-nap/entrypoint.sh:5]: imagename is legacy - this is not a thing any longer

File: debian/no-nap/entrypoint.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 #
 # This script launches nginx and the NGINX Controller Agent.
 #
-# If several instances use the same imagename, the metrics will
+# If several instances use the same image name, the metrics will
```
Comment (brianehlert) [inline debian/no-nap/entrypoint.sh:5]: imagename again

**Review by brianehlert — CHANGES_REQUESTED** [review]
imagename and imagename behavior is no longer a thing.  References should be removed.

**Review by azych — APPROVED** [review]

**Review by brianehlert — APPROVED** [review]

**Pull request merged**